imap/lmtp_sieve.c

```diff
@@ -1501,8 +1501,9 @@ static int sieve_imip(void *ac, void *ic, void *sc, void *mc,
         itip_strip_personal_data(comp);
     }
 
+    unsigned ical_support_flags = ICAL_SUPPORT_ALLOW_INVALID_IANA_TIMEZONE;
     cyrus_icalrestriction_check(itip);
-    if ((errstr = get_icalcomponent_errstr(itip)) &&
+    if ((errstr = get_icalcomponent_errstr(itip, ical_support_flags)) &&
```

Comment:
Why is here an indirection over a variable done, rather than passing a consant as second parameter?

Reply:
Right, there isn't really a good reason other than it's a left-over from refactoring. Removed.